network/src/storage.rs

```diff
@@ -0,0 +1,164 @@
+use crate::primitives::{Storage, StorageError};
+use async_trait::async_trait;
+use fs2::FileExt;
+use futures::stream::*;
+use tokio::{fs, io::AsyncWriteExt, task::spawn_blocking};
+
+pub struct StorageImpl {
+    lock_file: Option<std::fs::File>,
+    path: String,
+}
+
+#[async_trait]
+impl Storage for StorageImpl {
+    async fn create(storage_directory: &str) -> Result<(), StorageError> {
+        let _ = fs::remove_dir_all(storage_directory).await;
```

Comment:
This is not matched with the comment here: https://github.com/junha1/simperby/blob/3fcb5270c0249ee8764ff8eb890dd13aec51c095/network/src/primitives.rs#L10-L12

Either the implementation or the comment should be changed.